### DISCUSSION TITLE
K8S 组件工作目录

### DISCUSSION THREAD

**Discussion by RickJou** [labring/sealos#3358] [General] [answered]
默认 / 目录不会很大,那么对于containerd ,kubelet,等组件在Clusterfile中能否指定一个目录,比如: /data (另一块数据盘)作为工作目录呢.

**Comment by cuisongliu** [comment, marked as answer]
1. containerd使用环境变量可以做 https://docs.sealos.io/docs/lifecycle-management/advanced-guide/image-build-standardized (sealos run --env )
2. kubelet需要定制kubelet配置，参见 https://docs.sealos.io/docs/lifecycle-management/operations/run-cluster/gen-apply-cluster 可以修改kubelet的目录
3. sealos相关的路径  https://docs.sealos.io/docs/lifecycle-management/QA#%E6%96%87%E4%BB%B6%E5%92%8C%E7%9B%AE%E5%BD%95%E4%BD%8D%E7%BD%AE%E9%97%AE%E9%A2%98
